src/components/App.svelte

```diff
@@ -1,11 +1,7 @@
 <script>
-  export let name = 'Svelte';
+  export let name = 'Svelte'
```

Comment:
personally I don't like it when a semicolon is disabled.
But if prettier for svelt thinks differently, I'm okay with that